### PR TITLE
Products page

### PR DESCRIPTION
**Pull request by HarshnaNHaswani**
Please review! Kindly ignore cart and wishlist page commits

### CODE REVIEW

File: cart.html [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+
+
+      <li class="list-item">
+        <a href="/cart.html">Cart</a>
+      </li>
+
+
+
+
+
```
Comment (bharati-21) [inline cart.html:45]: Do look out at these empty spaces in case you forgot to remove.

**Review by bharati-21 — APPROVED** [review]
Looks good to be merged!

File: cart.html (hunk 1 above)
Comment (HarshnaNHaswani) [reply cart.html:45]: Thank you for the PR! I'll remove them

File: products.html [hunk 2]
```diff
@@ -0,0 +1,298 @@
...
+        autocomplete="name">
+    </div>
+    <ul class="list list-inline">
+
+
+      <li class="list-item">
+        <a href="/cart.html">Cart</a>
+      </li>
+
+
```
Comment (manujagobind) [inline products.html:42]: Please remove extra lines

**Pull request merged**